code/lib/builder-webpack5/src/index.ts

```diff
@@ -131,6 +133,11 @@ const starter: StarterFunction = async function* starterGeneratorFn({
 
   compilation = webpackDevMiddleware(compiler, middlewareOptions);
 
+  const previewResolvedDir = dirname(require.resolve('@storybook/preview/package.json'));
+  const previewDirOrigin = join(previewResolvedDir, 'dist');
+
+  router.use(`/sb-preview`, express.static(previewDirOrigin));
```

Comment:
Are we doing the equivalent thing in `builder-vite`?

Reply:
I will need to, indeed!

Reply:
@shilman mentioned I should try and get perf wins first, before spending too much time on something that might not get any perf gains.

Pre-bundling docs should be a huge perf win, but that is also extremely complex due to react and the weird nature of peerDeps around react & react-dom.